cmd/mesh/manifest-migrate.go

```diff
@@ -112,6 +131,57 @@ func translateFunc(values []byte, l *Logger) error {
 	return nil
 }
 
+func genICPSFromProfile(profile string, ver string) (string, error) {
```

Comment:
This function looks duplicated?

Reply:
this extracts some part from the existing function and delete some parts that are not applicable here

Reply:
Are they close enough to restructure the code to reuse it? Normally a bit of duplication is not a big deal for simple functions but the logic in here is not obvious and it will be hard to understand in the future how these two versions differ.

Reply:
I think we don't need this part of logic anymore based on our discussion.